pkg/cmd/upgrade/upgrade_boot.go

```diff
@@ -399,14 +466,13 @@ func (o *UpgradeBootOptions) setupGitConfig(dir string) error {
 }
 
 func (o *UpgradeBootOptions) excludeFiles(commit string) error {
-	excludedFiles := []string{"OWNERS"}
-	err := o.Git().CheckoutCommitFiles(o.Dir, commit, excludedFiles)
+	err := o.Git().CheckoutCommitFiles(o.Dir, commit, filesExcludedFromCherryPick)
 	if err != nil {
 		return errors.Wrap(err, "failed to checkout files")
 	}
-	err = o.Git().AddCommitFiles(o.Dir, "chore: exclude files from upgrade", excludedFiles)
+	err = o.Git().AddCommitFiles(o.Dir, "chore: exclude files from upgrade", filesExcludedFromCherryPick)
```

Comment:
I really have a bad feeling about this. Can we not just get rid of cherry-picking altogether?

Reply:
I just want to fix this one thing with `OWNERS` here and defer the root-and-branch work til later. =)